### ISSUE TITLE
FAST003 does not handle aliased path parameters

### ISSUE THREAD

**Issue by HenriBlacksmith** [astral-sh/ruff#16795]
### Summary

FAST003 rule raises errors when using aliased path parameters in an endpoint definition

# MRE

```python
from FastAPI import FastAPI, Path
from pydantic import BaseModel

ExampleIdPathParameter = Annotated[
    UUID4, Path(alias="exampleId", description="The unique identifier of an example resource.")
]

class Example(BaseModel):
     id: UUID4
     name: str

@router.get(
    "/examples/{exampleId}",
    name="example:read",
)
async def read_example(
    example_id: ExampleIdPathParameter,
) -> Example:
       return Example(id=example_id, name=f"example-{example_id}"
```

```
ruff check
```

### Version

ruff 0.11.0 (2cd25ef64 2025-03-14)

**Comment by HenriBlacksmith** [comment]
> Thanks. Ruff should detect aliased parameters ([#13263](https://github.com/astral-sh/ruff/issues/13263)) but we may miss aliased parameters when using `Annotated` with multiple elements

Sorry, I am fixing the MRE, it actually fails when the parameter is defined in another file
